docs/install.md

```diff
@@ -2,19 +2,19 @@
 
 Follow the steps below to install Codacy on an existing Kubernetes cluster using the provided cloud native Helm chart.
 
-1.  Create a Kubernetes namespace called `codacy` that will group all cluster resources related to Codacy.
```

Comment:
Our auto formaters work differently. Are you using remark? If so, we need to put a config file on the repo to make sure we format the same when we save

Reply:
@paulopontesm and @joaocosta-codacy, could you please describe briefly the setup that you're using?

I'm used to having [markdownlint](https://github.com/DavidAnson/markdownlint) as a Visual Studio Code extension to validate Markdown during the authoring of the files. This linter seems to be a bit more prescriptive on enforcing a certain Markdown style than remark-lint and it plays well with MkDocs (it's even the [linter used in the MkDocs project](https://github.com/mkdocs/mkdocs/blob/master/.markdownlintrc) itself).

However, since Codacy includes the remark-lint as the Markdown linter, I'm thinking that I should be using that one instead as well. So, let me know how you're currently using this linter so that I can adapt on my side as well as create a configuration file to enforce a few style options.

Reply:
I added a .remarkrc configuration file so that our Markdown style becomes consistent (https://github.com/codacy/chart/pull/293/commits/7963d611089f595d5853da112a1e5651c2837888). Hopefully this won't introduce too much noise in the diffs between files, if so, it may be better to remove the configuration file for now and only add it at a later time.